### PR TITLE
NAS-4246 cassettes optimization

### PR DESCRIPTION
**Pull request by hkad98**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-fdw/tests/execute/test_execute_compute_table.py [hunk 1]
```diff
@@ -2,17 +2,25 @@
...
+from tests_support.vcrpy_utils import VCR_MATCH_ON, CustomSerializerYaml, custom_before_request, custom_before_response
 
 from gooddata_fdw import GoodDataForeignDataWrapper
-from tests import VCR_MATCH_ON
 
 _current_dir = Path(__file__).parent.absolute()
 _fixtures_dir = _current_dir / "fixtures"
 
-gd_vcr = vcr.VCR(filter_headers=["authorization", "user-agent"], serializer="json", match_on=VCR_MATCH_ON)
+gd_vcr = vcr.VCR(
```
Comment (pcerny) [inline gooddata-fdw/tests/execute/test_execute_compute_table.py:12]: Create procedure which builds vcrpy with custom configuration.

**Review by pcerny — CHANGES_REQUESTED** [review]

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-fdw/tests/execute/test_execute_compute_table.py (hunk 1 above)
Comment (hkad98) [reply gooddata-fdw/tests/execute/test_execute_compute_table.py]: Done. Procedure created in tests-support package.

**Review by pcerny — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR145

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/131/ : SUCCESS in 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul-el7/131/ : FAILURE in 12s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/133/ : SUCCESS in 22s

**Pull request merged**